plugins/metrics/pkg/agent/http.go

```diff
@@ -132,3 +133,66 @@ func (s *HttpServer) handlePushRequest(c *gin.Context) {
 		c.Status(http.StatusServiceUnavailable)
 	}
 }
+
+//func (s *HttpServer) handleRemoteReadStart(c *gin.Context) {
```

Comment:
Do these commented out functions serve a purpose?

Reply:
nope, I definitely just commented those out when I was still tinkering and forgot about them. I'll delete them

Reply:
nope, I definitely just commented those out when I was still tinkering and forgot about them. I'll delete them now